node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -807,6 +816,17 @@ object NodeRuntime {
             )
         _ <- Time[F].sleep(conf.casper.casperLoopInterval.seconds)
       } yield ()
+      // Broadcast fork choice tips request if current fork choice is more then 3 minutes old.
+      // For why - look at updateForkChoiceTipsIfStuck method description.
+      updateForkChoiceLoop = {
+        implicit val cu = commUtil
+        implicit val ec = engineCell
+        implicit val bs = blockStore
+        for {
+          _ <- Running.updateForkChoiceTipsIfStuck(3.minutes)
+          _ <- Time[F].sleep(1.minutes)
```

Comment:
We should not hard-coded the parameter. Can you put it into configuration?

Reply:
@zsluedem 863a271